pkg/kp/dsstore/consul_store.go

```diff
@@ -326,6 +326,61 @@ func (s *consulStore) Watch(quitCh <-chan struct{}) <-chan WatchedDaemonSets {
 	return outCh
 }
 
+// Watch watches dsTree for changes and returns a blocking channel
+// where the client can read a WatchedDaemonSets object which contain
+// changed daemon sets
+func (s *consulStore) WatchAll(quitCh <-chan struct{}) <-chan WatchedDaemonSetList {
+	inCh := make(chan api.KVPairs)
+	outCh := make(chan WatchedDaemonSetList)
+	errCh := make(chan error, 1)
+
+	// Watch for changes in the dsTree and deletedDSTree
+	go consulutil.WatchPrefix(dsTree, s.kv, inCh, quitCh, errCh, time.Duration(0))
```

Comment:
In the interest of self-documenting code, can you change the duration to match what's actually happening here? (ie. change this to 250 ms)

```	if pause < 250*time.Millisecond {
		pause = 250 * time.Millisecond
	}
``` is near the top of WatchPrefix